### PR TITLE
[STRATCONN-4133] Add batch_size to actions-mixpanel

### PR DESCRIPTION
**Pull request by itsarijitray**
# Summary

Show batch_size for actions-mixpanel with minimum and maximum limits

[Mixpanel supports upto 2000 events per batch](https://developer.mixpanel.com/reference/track-event)

Warning!!
Do not merge this until the below dependent PRs are merged:

- https://github.com/segmentio/action-destinations/pull/2358
- https://github.com/segmentio/control-plane/pull/5209
- https://github.com/segmentio/control-plane/pull/5202

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [x] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/mixpanel/mixpanel-properties.ts [hunk 1]
```diff
@@ -389,6 +389,15 @@ export const eventProperties: Record<string, InputField> = {
...
     default: true
   },
+  batch_size: {
+    label: 'Batch Size',
+    description: 'Maximum number of events to include in each batch. Actual batch sizes may be lower.',
+    type: 'number',
+    required: false,
+    default: 1000,
+    minimum: 0,
+    maximum: 2000
```
Comment (varadarajan-tw) [inline packages/destination-actions/src/destinations/mixpanel/mixpanel-properties.ts:399]: How did we arrive at this min/max values?

Comment (varadarajan-tw) [reply packages/destination-actions/src/destinations/mixpanel/mixpanel-properties.ts:399]: Also, if someone sets batch size to 0, how would the destination behave? Would there be no batching?

Comment (itsarijitray) [reply packages/destination-actions/src/destinations/mixpanel/mixpanel-properties.ts:399]: For minimum, I have tried to constrain it to not allow any any negative numbers.

Comment (varadarajan-tw) [reply packages/destination-actions/src/destinations/mixpanel/mixpanel-properties.ts:399]: Does that mean a batch size of 0 could be misleading? If you set it to 1, does it work? If thats the case, should we set batch size min to 1?

Comment (itsarijitray) [reply packages/destination-actions/src/destinations/mixpanel/mixpanel-properties.ts]: Updated the tests

**Review by varadarajan-tw — APPROVED** [review]

**Pull request merged**